velox/substrait/tests/Velox2SubstraitPlanConverterTest.cpp

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <folly/Random.h>
+
+#include "velox/exec/tests/utils/OperatorTestBase.h"
+#include "velox/exec/tests/utils/PlanBuilder.h"
+#include "velox/vector/tests/VectorMaker.h"
+
+#include "velox/substrait/SubstraitToVeloxPlan.h"
+#include "velox/substrait/VeloxToSubstraitPlan.h"
+
+using namespace facebook::velox;
+using namespace facebook::velox::test;
+using namespace facebook::velox::exec;
+using namespace facebook::velox::exec::test;
+using namespace facebook::velox::substrait;
+
+class SubstraitVeloxPlanConvertorTest : public OperatorTestBase {
+ protected:
+  /// The function to make RowVectorPtr data which use the specific rowType_ and
+  /// batchSize. \param size the number of RowVectorPtr. \param batchSize The
+  /// batch Size of the data.
+  std::vector<RowVectorPtr> makeVector(int64_t size, int64_t batchSize) {
+    std::vector<RowVectorPtr> vectors;
+    std::mt19937 gen(std::mt19937::default_seed);
+    int64_t childSize = rowType_->size();
```

Comment:
This functions uses only number of columns from the rowType_. Hence, there is no need to use RowType in the first place, just pass the number of columns and document that this methods generates only INTEGER type columns.